bagger/src/files.py

```diff
@@ -15,10 +17,47 @@
 OBJECT_KEYS = set()
 
 
+def download_s3_object(b_number, source_bucket, source, destination):
+    try:
+        source_bucket.download_file(source, destination)
+    except ClientError:
+        logging.debug("Checking for case mismatch in ALTO file name")
+
+        # Sometimes the S3 objects have a case mismatch between the actual object
+        # and the ALTO reference.  In that case, we should try some alternative
+        # keys (by fiddling with the case) to see if one of the alternatives works.
+        #
+        # We use 'set()' to avoid trying the same key multiple times.
+        #
+        alternative_bnums = set([
+            b_number.replace("x", "X"),
+            b_number.replace("b", "B"),
+            b_number.replace("x", "X").replace("b", "B"),
+        ]) - set([b_number])
```

Comment:
Very small thing, I think if x is upper case b is also upper case, so you could increase the chances of a hit by:
```
[
            b_number.replace("x", "X"),
            b_number.replace("x", "X").replace("b", "B"),
            b_number.replace("b", "B"),
]
```